kythe/go/extractors/gcp/README.md

```diff
@@ -148,8 +148,9 @@ specify all of the above custom javac extraction logic.
 
 ### gcr.io/kythe-public/kzip-tools
 
-For now this is a simple wrapper around `zipmerge`, but will hopefully later
-contain other useful tools for dealing with kzip archives.
+For now this is a simple wrapper around
```

Comment:
It's not really a wrapper anymore, is it?

Reply:
Yea I was unsure about exact wording here.

I guess it's not a wrapper.  The fact that gcb wonkiness means I need to 'bash -c ...' stuff isn't so much an artifact of the docker image itself, but rather gcb.  Or my lack of knowledge on how to properly wildcard match stuff in gcb.